dagmc_stats.py

```diff
@@ -153,51 +154,111 @@ def get_surfaces_per_volume(my_core, entityset_ranges):
-        s = 0
-        coord_list = []
+def get_tri_side_length(my_core, tri):
+    """
+    get side lengths of triangle
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    tri : triangle
+
+    outputs
+    -------
+    side_lengths : (list)side lengths of triangle
+    """
+    side_lengths = []
+    s = 0
+    coord_list = []
+
+    verts = list(my_core.get_adjacencies(tri, 0))
+
+    for vert in verts:
+        coords = my_core.get_coords(vert)
+        coord_list.append(coords)
 
-        verts = list(my_core.get_adjacencies(triangle, 0))
+    for side in range(3):
+        side_lengths.append(np.linalg.norm(coord_list[side]-coord_list[side-2]))
+        # The indices of coord_list includes the "-2" because this way each side will be matched up with both
+        # other sides of the triangle (IDs: (Side 0, Side 1), (Side 1, Side 2), (Side 2, Side 0))
+    return side_lengths
 
-        for vert in verts:
-            coords = my_core.get_coords(vert)
-            coord_list.append(coords)
 
-        for side in range(3):    
-            side_lengths.append(np.linalg.norm(coord_list[side]-coord_list[side-2]))
-            # The indices of coord_list includes the "-2" because this way each side will be matched up with both
-            # other sides of the triangle (IDs: (Side 0, Side 1), (Side 1, Side 2), (Side 2, Side 0))
-      
+def get_triangle_aspect_ratio(my_core, meshset, geom_dim):
+    """
+    Gets the triangle aspect ratio (according to the equation: (abc)/(8(s-a)(s-b)(s-c)), where s = .5(a+b+c).)
+
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    meshset : a meshset containing a certain part of the mesh
+    geom_dim : a MOAB Tag that holds the dimension of an entity.
+
+    outputs
+    -------
+    t_a_r : (list) the triangle aspect ratios for all triangles in the meshset
+    """
+    tris = get_tris(my_core, meshset, geom_dim)
+    t_a_r = []
+
+    for tri in tris:
+        side_lengths = get_tri_side_length(my_core, tri)
         s = .5*(sum(side_lengths))
         top = np.prod(side_lengths)
         bottom = 8*np.prod(s-side_lengths)
         t_a_r.append(top/bottom)
 
     return t_a_r
+
+def get_area_triangle(my_core, meshset, geom_dim):	
+    """
+    Gets the triangle area (according to the equation: sqrt(s(s - a)(s - b)(s - c)), where s = (a + b + c)/2)
+    
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    meshset : a meshset containing a certain part of the mesh
+    geom_dim : a MOAB Tag that holds the dimension of an entity.
+    
+    outputs
+    -------
+    area : (list) the triangle areas for all triangles in the meshset
+    """
+    
+    area = []
+    tris = get_tris(my_core, meshset, geom_dim)
+
+    for tri in tris:
+        side_lengths = get_tri_side_length(my_core, tri)
+        # sqrt(s(s - a)(s - b)(s - c)), where s = (a + b + c)/2
+        s = sum(side_lengths)/2
+        s = np.sqrt(s * np.prod(s - side_lengths))
+        area.append(s)
```

Comment:
same as my above comment and what I think @gonuke is suggesting. These lines could be moved into their own function that uses `side_lengths` as input and returns `area` to be appended in the list.